net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Threads Max</fielddescr>
+			<fieldname>threads_max</fieldname>
+			<description>
+				<![CDATA[
+					This option allows you to set the maximum number of threads that will<br/>
+					execute at once.  Defaults to 10<br/>
+				]]>
+			</description>
+			<default_value>10</default_value>
+			<type>input</type>
+			<size>3</size>
+		</field>
+		<field>
+			<fielddescr>Date Format</fielddescr>
+			<fieldname>date_format</fieldname>
+			<description>The date format for $x in strftime() format.</description>
+			<default_value>%a %b %e %Y</default_value>
+			<type>input</type>
+			<size>20</size>
+			<required>true</required>
+		</field>
+		<field>
+			<fielddescr>Time Format</fielddescr>
+			<fieldname>time_format</fieldname>
+			<description>The time format for $X in strftime() format.</description>
+			<default_value>%H:%M:%S</default_value>
+			<type>input</type>
+			<size>20</size>
+			<required>true</required>
+		</field>
+		<field>
+			<fielddescr>Date Time Format</fielddescr>
+			<fieldname>date_time_format</fieldname>
+			<description>
+				<![CDATA[
+					The date time format in strftime() format for the date/time when logging<br/>
+					to standard output, snmptt log files (log_file) and the unknown log file<br/>
+					(unknown_trap_log_file). Defaults to localtime().<br/>
+				]]>
+			</description>
+			<default_value>%a %b %e %Y %H:%M:%S</default_value>
+			<type>input</type>
+			<size>20</size>
+			<required>true</required>
+		</field>
+		<field>
+			<name>Daemon Mode</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Sleep</fielddescr>
+			<fieldname>sleep</fieldname>
+			<description>Amount of time in seconds to sleep between processing spool files.</description>
+			<default_value>5</default_value>
+			<type>input</type>
+			<size>3</size>
+			<required>true</required>
+		</field>
+		<field>
+			<fielddescr>Use Trap Time</fielddescr>
+			<fieldname>use_trap_time</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have SNMPTT use the time that the trap was processed by SNMPTTHANDLER<br/>
+					Set to 0 to have SNMPTT use the time the trap was processed.  Note:  Using 0 can<br/>
+					result in the time being off by the number of seconds used for 'sleep'<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Keep Unlogged Traps</fielddescr>
+			<fieldname>keep_unlogged_traps</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to have SNMPTT erase the spooled trap file after it attempts to process<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.